carma/launch/environment.launch.py

```diff
@@ -307,7 +313,7 @@ def generate_launch_description():
                         ("external_objects_viz", "fused_external_objects_viz")
                     ],
                     parameters=[object_visualizer_param_file, vehicle_config_param_file,
-                                {'pedestrian_icon_path': 'file:///', vehicle_calibration_dir, '/visualization_meshes/pedestrian.stl'}
+                                {'pedestrian_icon_path': ['file:///', vehicle_calibration_dir, '/visualization_meshes/pedestrian.stl']}
```

Comment:
This fix is in stingray. Without it the platform shuts down.

Reply:
Looks like it exists in stingray but not in develop. Created a PR for this here https://github.com/usdot-fhwa-stol/carma-platform/pull/2567